### PR TITLE
Clean project hierarchy & Settings

### PR DESCRIPTION
**Pull request by amfathi**
Working on refactoring the code bit by bit:

This PR is not intended to touch any functionality, but instead separating layers and dependancies for easier refactoring and debugging in the future.

Steps I took:
- Starting by project hierarchy.
- Supported iOS version.
- ContentView.
- SignView.
- SettingsView.

NOTE: This is a draft PR for early reviews, feel free to add comments, will convert to not-draft once this step is done.
NOTE: Also feel free to download and test the app every now and then and create issue for any currently working functionality got broken during this refactors

### CODE REVIEW

File: NthKey.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -690,7 +690,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
```
Comment (Sjors) [inline NthKey.xcodeproj/project.pbxproj:773]: What's the reason for lowering the target? Since the app is for early adopters I prefer to start with the newest SDK, to avoid legacy baggage. We can add compatibility for older devices in the future.

File: NthKey.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -708,16 +708,16 @@
...
 				INFOPLIST_FILE = "$(SRCROOT)/NthKey/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 14.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 0.0.8;
-				PRODUCT_BUNDLE_IDENTIFIER = com.nthkey.app;
+				PRODUCT_BUNDLE_IDENTIFIER = com.nthkey.app.test;
```
Comment (Sjors) [inline NthKey.xcodeproj/project.pbxproj:800]: Where does bundle identifier change take effect? Just on the simulator and device when doing development? Or also on TestFlight?

File: NthKey.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		1F6A2E586F1545C3C5EFA149 /* Pods_NthKey.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CA4B65438CE45530547E491B /* Pods_NthKey.framework */; };
+		4AB1EA3425AB194F00EE6832 /* AddressessView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AB1EA3325AB194F00EE6832 /* AddressessView.swift */; };
```
Comment (Sjors) [inline NthKey.xcodeproj/project.pbxproj:11]: Typo: `AddressesView.swift`

File: NthKey.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -9,6 +9,9 @@
 /* Begin PBXBuildFile section */
 		1F6A2E586F1545C3C5EFA149 /* Pods_NthKey.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CA4B65438CE45530547E491B /* Pods_NthKey.framework */; };
 		4AB1EA3425AB194F00EE6832 /* AddressessView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AB1EA3325AB194F00EE6832 /* AddressessView.swift */; };
+		4AB1EA7225AB375200EE6832 /* QRCodeBuilder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AB1EA7125AB375200EE6832 /* QRCodeBuilder.swift */; };
+		4AB1EA7A25AB3B5D00EE6832 /* AnnonceView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AB1EA7925AB3B5D00EE6832 /* AnnonceView.swift */; };
```
Comment (Sjors) [inline NthKey.xcodeproj/project.pbxproj:13]: Typo: `AnnounceView`

File: NthKey/Model/KeychainEntropyItem.swift [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
-    can be converted to a 12-24 word BIP39 mnemonic.
-*/
+ Copyright © 2016 Apple Inc. All Rights Reserved.
+ Copyright © 2019 Purple Dunes. Distributed under the MIT software
+ license, see the accompanying file LICENSE.md
+ 
+ Abstract:
+ A struct for accessing BIP39 entropy Data in the keychain. This entropy
+ can be converted to a 12-24 word BIP39 mnemonic.
+ */
```
Comment (Sjors) [inline NthKey/Model/KeychainEntropyItem.swift:9]: These whitespace changes clutter up the commit, so would be good to put those in a separate commit (maybe 1 to cover all files). Are you using a tool for this?

**Review by Sjors — COMMENTED** [review]
Thanks! Changes look good, but I left a few comments to make it more readable. I always like the commit history to be clear, so it's easier to use `git blame` on future bugs.

Can you split `initial commit for clean up`? The changes seem unrelated.

If it's possible to split `Refactoring SettingsView.` from `Rearrange project heirarchy.` that would be nice too, but perhaps the change is too related.

File: NthKey.xcodeproj/project.pbxproj (hunk 1 above)
Comment (amfathi) [reply NthKey.xcodeproj/project.pbxproj:773]: My device was not updated to 14.2 yet, and 13 seems pretty reasonable major release to work with.

File: NthKey.xcodeproj/project.pbxproj (hunk 2 above)
Comment (amfathi) [reply NthKey.xcodeproj/project.pbxproj:800]: This will be reverted (Y)

File: NthKey/Model/KeychainEntropyItem.swift (hunk 5 above)
Comment (amfathi) [reply NthKey/Model/KeychainEntropyItem.swift:9]: This is xcode default indentation tool, I'll separate that commit.

File: NthKey.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Sjors) [reply NthKey.xcodeproj/project.pbxproj:773]: Ok, that's fine then. LibWallySwift goes back to 12 I believe, so this shouldn't be an issue.

File: NthKey.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -251,31 +332,23 @@
...
 		A27180B423A293BC00E9F443 /* View */ = {
 			isa = PBXGroup;
 			children = (
-				A27180B523A293EA00E9F443 /* AddressView.swift */,
 				A206E80B238D65F300786824 /* ContentView.swift */,
-				A285870C23AD31C600930566 /* Destination.swift */,
-				4AB1EA3325AB194F00EE6832 /* AddressessView.swift */,
-				A2E9E37E23ACF1140049645D /* SignView.swift */,
-				A27D4BF523A2B4F30055B0B8 /* SettingsView.swift */,
+				4AB1EA3B25AB314700EE6832 /* Adresses */,
```
Comment (amfathi) [inline NthKey.xcodeproj/project.pbxproj:344]: Typo: `Addresses`
